src/generate-signed-data.tsx

```diff
@@ -0,0 +1,26 @@
+import { signSmartContractData } from "@wert-io/widget-sc-signer";
+type SignSmartContractDataParameters = Parameters<
+  typeof signSmartContractData
+>[0];
+type PartialBy<T, K extends keyof T> = Omit<T, K> & Partial<Pick<T, K>>;
```

Comment:
I had to make the 'network' field optional, because it's actually [required](https://github.com/wert-io/widget-sc-signer/blob/master/index.d.ts#L5C12-L5C12) by the signer, but in its readme, it says that it's [optional](https://github.com/wert-io/widget-sc-signer#options). If it is actually required, I can remove this extra typing